api/docs/building.dox

```diff
@@ -38,6 +38,15 @@
 
 If you don't need to modify DR and just need a recent build, you can download [weekly package builds](@ref page_weekly_builds).
 
+The official list of environments for which we have complete or in-progress support are the ones on our [Github
```

Comment:
This needs to be reconciled with `sec_limit_platforms` which has lists of versions of each OS which are supposedly supported; the rendered contents are at https://dynamorio.org/release_notes.html#sec_limit_platforms.

A link to that section should be included here, and that section itself should likely be updated, and some wording explaining the level of support for CI workflows vs "should work" on the limit list.